### DISCUSSION TITLE
filter server tags not saving after change

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#14557] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![изображение](https://github.com/user-attachments/assets/30a6e1dc-2265-4fed-b6cd-be174bb4d702)
dropdown menus are not saving config

### Reproduction steps

choose any filter variants in dropdown menu, exit server browser, reopen server browser, observe unsaved changes

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Seems to have been fixed for the upcoming release (and works with no issues in the current unstable build). Sorry for the inconvenience!
